powerlevel9k.zsh-theme

```diff
@@ -429,10 +429,12 @@ prompt_dir() {
   fi
 
```

Comment:
You are almost there. Imagine this segment has three states:

1. Being at the root of your Home folder
    This is the first condition. So we want to match only that. Looks good in your changeset.

2. Being in a subdirectory of your Home folder
    The condition looks good (matching a path that _begins_ with a tilde.

3. Being anywhere else